<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPClient</class>
 <widget class="QWidget" name="TCPClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCPClient</string>
  </property>
  <widget class="QPushButton" name="btn_salir">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_message">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_ip">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_port">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
